app/frontend/actions.js

```diff
@@ -104,6 +112,7 @@ module.exports = ({setState, getState}) => {
       const sendAddress = {fieldValue: ''}
       const sendResponse = ''
       const usingTrezor = cryptoProvider === 'trezor'
```

Comment:
I would rather generalize this to "usingHwWallet", so no additional "or"-ing would be needed in the code below